CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ NEW FEATURES:
     - `provider::terraform::encode_expr` - Encode an arbitrary expression into a string with valid OpenTofu syntax.
 - Added support for S3 native locking ([#599](https://github.com/opentofu/opentofu/issues/599))
 - Backend `pg` now allows the `table_name` and `index_name` to be specified. This enables a single database schema to support multiple backends via multiple tables. ([#2465](https://github.com/opentofu/opentofu/pull/2465))
+- New `opentofu` CLI options with `-target-file` and `-exclude-file` allow for simplified workflows for targeting many or frequently reused resources.
```

Comment:
We don't tend to mention `opentofu` in the changelog as it's implied.  If we were, we would probably call it `tofu` as that's the typical binary name.